src/components/MainNav/CurrentApp/AppContextDropdown.js

```diff
@@ -29,7 +29,7 @@ class AppContextDropdown extends React.Component {
         open={open}
         label={
           <>
-            {selectedApp.displayName}
+            {selectedApp?.displayName}
```

Comment:
Thanks! I wonder if this was a new glitch introduced by React.lazy, since previously all apps would be loaded, but that is no longer the case. (CC: @mkuklis)